VS2008 build issue with protobuf 3.5.0

Ran into the following issue when trying to build `protobuf` 3.5.0 where `map::at` was used. The [`map::at` method only exists in C++11]( http://en.cppreference.com/w/cpp/container/map/at ). As such it cannot compile with a non-C++11 compiler like VS2008.

```
C:\bld\protobuf_1513642428254\work\protobuf-3.5.0\src\google\protobuf\util\field_mask_util.cc(377) : error C2039: 'at' : is not a member of 'std::map<_Kty,_Ty>'
        with
        [
            _Kty=std::string,
            _Ty=google::protobuf::util::`anonymous-namespace'::FieldMaskTree::Node *
        ]
NMAKE : fatal error U1077: 'C:\PROGRA~2\MICROS~1.0\VC\bin\cl.exe' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\BIN\nmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\BIN\nmake.exe"' : return code '0x2'
Stop.
```

ref: https://ci.appveyor.com/project/conda-forge/protobuf-feedstock/build/job/o85bad6hfjhvyxvu

--- a/src/google/protobuf/util/field_mask_util.cc
+++ b/src/google/protobuf/util/field_mask_util.cc
@@ -374,7 +374,7 @@ void FieldMaskTree::RemovePath(const string& path,
       }
     }
     if (ContainsKey(node->children, parts[i])) {
-      node = node->children.at(parts[i]);
+      node = node->children[parts[i]];
       if (field_descriptor->cpp_type() == FieldDescriptor::CPPTYPE_MESSAGE) {
         current_descriptor = field_descriptor->message_type();
       }

